examples/Console/otlp-collector-example/config.yaml

```diff
@@ -12,7 +12,7 @@ receivers:
 
 exporters:
   logging:
-    loglevel: debug
+    verbosity: Detailed
```

Comment:
```suggestion
    verbosity: detailed
```

Reply:
Just to stay consistent with the casing used in existing code. https://github.com/open-telemetry/opentelemetry-dotnet/blob/87b970b07a2f8172d1112e4b0e3eac59999c572b/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/otel-collector-config.yaml#L27-L29